### DISCUSSION TITLE
Nginx Connection refused after update from RC3

### DISCUSSION THREAD

**Discussion by shipler** [Security-Onion-Solutions/securityonion#1546] [Unsupported Versions] [answered]
After updating from RC3 to the latest version using local iso mounting, the nginx is getting the following in the so/log/nginx/error.log -

> connect() failed (111: Connection refused) while connecting to upstream, client x.x.x.x, server: _, request: "XXXXX"

When accessing the master web port i get "502 bad gateway" error, also tried restarting the OS but the error persists

**Comment by TOoSmOotH** [comment]
What does so-status show?

**Reply by shipler** [reply]
so-soc ERROR, and now the nginx error is "Authorization Required 401". I'm going to reinstall the cluster...

**Reply by shipler** [reply]
Reinstalled the Master server, same error. so-soc not starting. and there's no logfile in the log/soc folder. how can I debug this?
The docker ps command not showing so-soc image

**Reply by shipler** [reply]
docker logs so-soc showing error:

> Error: Unable to read configuration file 'sensoroni.json' [invalid charachter '\n' in string literal]
> error Syntax error reading json object error=invalid charachter '\n' in string literal offset=814

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/commit/4a9fcfb8cf9ae9631d77b7acd7a5451efd22496b should fix the issue. Just make the same change. This will be fixed in 2.3.1

**Reply by shipler** [reply]
Yep, same error. Thanks!

**Comment by ColeVan** [comment]
I'm getting the exact same errors when trying to access the SOC page. Is there a way to remedy this? 

There are no log files in the soc directory to troubleshoot. nginx error.log shows the same error as above.

**Reply by TOoSmOotH** [reply]
Run soup. This is fixed in 2.3.1

**Reply by ColeVan** [reply]
Will this also update the nodes connected to the existing manager if soup is ran?

**Reply by TOoSmOotH** [reply]
yes

**Reply by ColeVan** [reply]
Once I update and try to login I get this error. Any thoughts? 

**The login request has expired. Redirect the user to the login endpoint to initialize a new session.**

And then when you go to login with the cred I get this error

**{"error":{"code":400,"status":"Bad Request","reason":"CSRF token is missing or invalid.","message":"The request was malformed or contained invalid parameters"}}**

**Reply by TOoSmOotH** [reply]
Please open a new discussion for your issue. It is unrelated to this one.
